crates/macro/ui-tests/start-function.rs

```diff
@@ -0,0 +1,12 @@
+extern crate wasm_bindgen;
+
+use wasm_bindgen::prelude::*;
+
+#[wasm_bindgen(start)]
+pub fn foo() {}
+
+#[wasm_bindgen(start)]
+pub fn foo2(x: u32) {}
+
+#[wasm_bindgen(start)]
+pub fn foo3<T>() {}
```

Comment:
Would be good to have a test for two otherwise valid start functions, to ensure that we properly error and don't accidentally just take the first one or something.

Reply:
Ah unfortunately we don't have a great place to put such tests today, but I've filed https://github.com/rustwasm/wasm-bindgen/issues/1058 to track this!